agents/appmotion_motor/appmotion_motors_agent.py

```diff
@@ -138,29 +166,28 @@ def move_axis_to_position(self, session, params=None):
             lin_stage (bool): True if the specified motor is for the linear
                 stage, False if not (default True)
         """
-
-        @ocs_agent.param('lin_stage', default=True, type=bool)
-        @ocs_agent.param('motor', default=1, type=int)
-        @ocs_agent.param('pos_is_inches', default=False, type=bool)
-        @ocs_agent.param('pos', default=0, type=float)
-        self.move_status = self.is_moving(motor)
-        if self.move_status:
-            return False, "Motors are already moving."
-
+        
+        self.move_status = self.is_moving(motor)[1][1]
         with self.lock.acquire_timeout(1, job=f'move_axis_to_position_motor{motor}') as acquired:
+            if self.move_status:
+                return False, "Motors are already moving."
             if not acquired:
                 self.log.warn(
                     f"Could not move motor{motor} because lock held by {self.lock.job}")
                 return False, "Could not acquire lock"
             self.motors.move_axis_to_position(motor, pos, pos_is_inches, lin_stage)
```

Comment:
Right, so I imagine here is where we'd handle which motor to send the commands to. And do this for all functions that require a command be sent to a specific motor.

for example:
if motorID == 1,
   call self.**motor1**.move_axis_to_position(...)

Reply:
made requested changes